aeron-archive/src/main/java/io/aeron/archive/ReplicationSession.java

```diff
@@ -57,6 +57,7 @@ class ReplicationSession implements Session, RecordingDescriptorConsumer
     private long srcStopPosition = NULL_POSITION;
     private long srcRecordingPosition = NULL_POSITION;
     private final long dstStopPosition;
+    private final boolean dstRecordingIsEmpty;
```

Comment:
`isDestinationRecordingEmpty` is a better. Booleans starting with `isX` or `hasX` is common coding standard.

Reply:
Originally I added a method name `isDestinationRecordingEmpty` to the `RecordingSummary` and used in several places. But then the only consumer of it was the `ReplicationSession` so I left the poorly named variable instead. Fixed now.